conans/client/rest/remote_credentials.py

```diff
@@ -82,8 +80,8 @@ def _get_env(remote, user):
             ConanOutput().info("Got password '******' from environment")
         return user, passwd
 
+
```

Comment:
Pep8 says 2 blank lines here (enabling some linter in your IDE will make this highlighted)